django/test/selenium.py

```diff
@@ -147,6 +174,36 @@ def mobile_size(self):
         with ChangeWindowSize(360, 800, self.selenium):
             yield
 
+    @contextmanager
+    def rtl(self):
+        get_language_bidi_mock = mock.patch(
+            "django.utils.translation.get_language_bidi", return_value=True
+        )
+        get_language_bidi_mock.start()
```

Comment:
Can we use one of bidirectional languages instead of this mocking?
```python
with override_settings(LANGUAGE_CODE="he"):
    ...
```
or
```python
with override_settings(LANGUAGE_CODE=settings.LANGUAGES_BIDI[-1]):
    ...
```

Reply:
This works 🎉 
It does mean that in tests when there is any selection based off link text `find_element(By.LINK_TEXT, "Example")` - it doesn't work out of the box 🤔 
What do you think? Is it better because the language is realistic? Or is it nice to keep it in english and have it go the other side via the mocking?